test/e2e/common/configmap_volume.go

```diff
@@ -622,17 +621,14 @@ func doConfigMapE2EWithoutMappings(f *framework.Framework, uid, fsGroup int64, d
 
 	if defaultMode != nil {
 		pod.Spec.Volumes[0].VolumeSource.ConfigMap.DefaultMode = defaultMode
-	} else {
-		mode := int32(0644)
-		defaultMode = &mode
 	}
 
-	modeString := fmt.Sprintf("%v", os.FileMode(*defaultMode))
+	fileModeRegexp := framework.GetFileModeRegex("/etc/configmap-volume/data-1", defaultMode)
 	output := []string{
-		"content of file \"/etc/configmap-volume/data-1\": value-1",
-		"mode of file \"/etc/configmap-volume/data-1\": " + modeString,
+		"content of file \"\\/etc\\/configmap-volume\\/data-1\": value-1",
```

Comment:
nit: you can avoid some of the escaping here by using raw strings, e.g.
```golang
`content of file "\/etc\/configmap-volume\/data-1": value-1`
```

Reply:
also, in `getFileAttributeRegex`, it reads `mode of Windows file`. should there be a `Windows` here too?

Reply:
Done.

The file content is still the same, so there is no need to change the expected output related to the file content. This patch only addresses the differences of default file permissions between Linux and Windows.